src/doc/yaml/cms.yaml

```diff
@@ -0,0 +1,565 @@
+          required: false
+          schema:
+            type: array
+            items:
+              type: string
+        - name: transactionId
+          in: query
+          description: The id of the transaction
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Read successful
+          $ref: '#/components/responses/CredentialProfilesResponse'
+        '400':
+          description: Bad request
+        '403':
+          description: Read not allowed
+        '404':
+          description: Unknown record
+        '500':
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+
+
+components:
+  responses:
+    CredentialRequestResponse:
+      description: The response for a credential request
+      content:
+        application/json:
+          schema:
+            allOf:
+              - $ref: '#/components/schemas/CredentialRequestId'
+              - $ref: '#/components/schemas/PersonId'
+              - $ref: '#/components/schemas/CredentialRequest'      
+
+    CredentialResponse:
+      description: The response for a credential request
+      content:
+        application/json:
+          schema:
+            allOf:
+              - $ref: '#/components/schemas/CredentialId'
+              - $ref: '#/components/schemas/PersonId'
+              - $ref: '#/components/schemas/Credential'
+
+    CredentialsResponse:
+      description: The response for a credential request
+      content:
+        application/json:
+          schema:
+            type: array
+            items:
+              allOf:
+                - $ref: '#/components/schemas/CredentialId'
+                - $ref: '#/components/schemas/PersonId'
+                - $ref: '#/components/schemas/Credential'
+
+    CredentialProfilesResponse:
+      description: The response for a credential profile
+      content:
+        application/json:
+          schema:
+            type: array
+            items:
+              allOf:
+                - $ref: '#/components/schemas/CredentialProfileId'
+                - $ref: '#/components/schemas/CredentialProfile'
+
+  schemas:
+    Error:
+      type: object
+      required:
+        - code
+        - message
+      properties:
+        code:
+          description: Error code
+          type: integer
+          format: int32
+        message:
+          description: Error message
+          type: string
+
+    PersonId:
+        type: object
+        properties:
+          personId:
+            type: string
+            description: The unique id of person that the credential request is for
+
+    CredentialRequestId:
+      type: object
+      properties:
+        credentialRequestId:
+          type: string
+          description: The unique id of this credential request
+
+    CredentialRequest:
```

Comment:
I believe the CredentialRequest should contain all data needed to make the credential. personId is part of it but not enough, we need also the names, date of birth, maybe face, etc.

Reply:
Updated.